app/models/category.rb

```diff
@@ -0,0 +1,5 @@
+class Category < ApplicationRecord
+  validates :name, presence: true, null: false
```

Comment:
You don't need `null: false`
Just `validates :name, presence: true`
`null: false` you add in `migration` on the necessary field